### DISCUSSION TITLE
inquiry regarding the multi mapper option

### DISCUSSION THREAD

**Discussion by Marwansha** [alexdobin/STAR#1941] [Q&A] [answered]
Hey,
I have a small  and probably naive question regarding the StarSolo workflow, I am trying to run it to obtain results similar to cell ranger v4 but add the mulitmapper option too #--soloMultiMappers EM option

so I utilized this option as recommended  for Matching CellRanger 4.x.x and 5.x.x results which is 
--clipAdapterType CellRanger4 --outFilterScoreMin 30
--soloCBmatchWLtype 1MM_multi_Nbase_pseudocounts --soloUMIfiltering MultiGeneUMI_CR --soloUMIdedup 1MM_CR  
my question is 
does the **soloUMIfiltering MultiGeneUMI_CR_** contradict with the multi mapper option , and what UMI filtering would be suitable in case its is

thanks

**Comment by alexdobin** [comment]
Hi Marwansha,

this is a good question. `--soloUMIfiltering MultiGeneUMI_CR` will work fine with EM options. It will still produce the unique-gene matrix that very similar to CR, and the EM matrix with multi-gene reads.

**Reply by Marwansha** [reply]
thanks for your reply

so i would have 2 different count matrix one with unique gene and other one with multi gene count integrated?

did i get it right?

Thanks alot

**Comment by alexdobin** [comment, marked as answer]
Yes, one is unique-gene only, and the other unique+multiple.
